include/seqan/gff_io/gff_io_base.h

```diff
@@ -436,6 +437,14 @@ void readRecord(GffRecord & record, CharString & buffer, TFwdIterator & iter)
             break;
         }
     }
+
+    // The last line might be a "### directive" specifically in GFF3
+    // Need to skip it to avoid another call of readRecords
+    skipUntil(iter, NotFunctor<IsWhitespace>());  //skip empty lines
+    while (!atEnd(iter) && value(iter) == '#')
```

Comment:
I am not fully aware of the specifications, but lines starting with `#` can always be ignored?

Reply:
Yes they can be ignored safely.

Reply:
Good then :smile: